hs-ucan/library/Web/UCAN/Types.hs

```diff
@@ -8,8 +9,12 @@ module Web.UCAN.Types
 
   -- * reexports
 
+  , module Web.UCAN.Capabilities.Types
+  , module Web.UCAN.Claims.Types
   , module Web.UCAN.Header.Types
+  , module Web.UCAN.Nonce.Types
   , module Web.UCAN.RawContent
+  , module Web.UCAN.Proof.Types
```

Comment:
This module go quite big, so I split it into multiple smaller ones.

Reply:
Yes this totally makes sense. I had tried to break it up previously, but that required abstracting over a bunch of fields... which now _need_ to be abstracted because of higher generality, so 👍👍👍